README.md

```diff
@@ -37,10 +37,11 @@ docker-compose up
 ## Environment Variables
 
 | Variable                    | Description                                                                                                                                                  | Default                            |
-|-----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------|
+| --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------- |
 | `BASE_PATH`                 | Set this variable if you run wireguard-ui under a subpath of your reverse proxy virtual host (e.g. /wireguard))                                              | N/A                                |
-| `BIND_ADDRESS`              | The addresses that can access to the web interface and the port                                                                                              | 0.0.0.0:80                        |
+| `BIND_ADDRESS`              | The addresses that can access to the web interface and the port                                                                                              | 0.0.0.0:80                         |
 | `SESSION_SECRET`            | The secret key used to encrypt the session cookies. Set this to a random value                                                                               | N/A                                |
+| `SUBNET_RANGES`             | The list of address subdivision ranges. Format: `SR Name: 10.0.1.0/24; SR2: 10.0.2.0/24, 10.0.3.0/24` Each CIDR must be inside one of the server interfaces. | N/A                                |
```

Comment:
Just leave the important stuff. 

Test also to drop that example string into the docker-compose.yaml file. Error. 

Test it on https://www.yamllint.com/

```yaml
...
    environment:
      - SUBNET_RANGES=SR Name:10.0.1.0/24; SR2:10.0.2.0/24,10.0.3.0/24
...
```

Clean it up so that it works.

Reply:
Because that's where it will also get used. Why make it harder for users to
get started? Friction.